### PR TITLE
Disable bot to add labels based on platforms

### PR DESCRIPTION
**Pull request by clee2000**
Some platforms have corresponding labels so add labels based on platform if there is only one set of relevant labels.  

Check labels when issue is opened or edited and correct labels if needed.

### CODE REVIEW

File: torchci/lib/bot/verifyDisableTestIssueBot.ts [hunk 1]
```diff
@@ -10,17 +12,17 @@ export const disabledTestIssueTitle = new RegExp(
...
-  "mac",
-  "macos",
-  "rocm",
-  "slow",
-  "win",
-  "windows",
-  "dynamo",
-  "inductor",
+export const supportedPlatforms = new Map([
+  ["asan", undefined],
```
Comment (huydhn) [inline torchci/lib/bot/verifyDisableTestIssueBot.ts:16]: Overall LGTM, although I think it's nicer to have the value here as an array of label instead for flexibility, i.e.

```
export const supportedPlatforms = new Map([
  ["asan", []],
  ["linux", []],
  ["mac", ["module: macos"]],
  ["macos", ["module: macos"]],
  ["rocm", ["module: rocm"]],
  ["slow", []],
  ["win", ["module: windows"]],
  ["windows", ["module: windows"]],
  ["dynamo", ["oncall: pt2"]],
  ["inductor", ["oncall: pt2"]],
]);
```

The rest could be built around this.  Thoughts?

**Review by huydhn — APPROVED** [review]

File: torchci/test/disableFlakyBot.test.ts [hunk 2]
```diff
@@ -668,7 +675,7 @@ describe("Disable Flaky Test Bot Unit Tests", () => {
 
     const { labels, additionalErrMessage } =
       await disableFlakyTestBot.getTestOwnerLabels(flakyTestA);
-    expect(labels).toEqual(["module: unknown"]);
+    expect(labels).toEqual(["module: unknown", "module: windows", "triaged"]);
```
Comment (janeyx99) [inline torchci/test/disableFlakyBot.test.ts:678]: module: unknown should NOT also occur with another module

File: torchci/lib/bot/verifyDisableTestIssueBot.ts [hunk 3]
```diff
@@ -10,17 +12,17 @@ export const disabledTestIssueTitle = new RegExp(
...
+  ["asan", []],
+  ["linux", []],
+  ["mac", ["module: macos"]],
+  ["macos", ["module: macos"]],
+  ["rocm", ["module: rocm"]],
+  ["slow", []],
+  ["win", ["module: windows"]],
+  ["windows", ["module: windows"]],
+  ["dynamo", ["oncall: pt2"]],
+  ["inductor", ["oncall: pt2"]],
```
Comment (janeyx99) [inline torchci/lib/bot/verifyDisableTestIssueBot.ts:25]: the processes later will add "triaged" for the "module" ones right?

Comment (huydhn) [reply torchci/lib/bot/verifyDisableTestIssueBot.ts:25]: I think `triaged` label could all be added for `module` and `oncall` right here now that it's a list of labels

Comment (janeyx99) [reply torchci/lib/bot/verifyDisableTestIssueBot.ts:25]: No need to add triaged for oncall labels btw

**Pull request merged**